Gordon360/Controllers/RecIM/TeamsController.cs

```diff
@@ -186,5 +186,32 @@ public ActionResult<IEnumerable<TeamInviteViewModel>> GetTeamInvites()
             var username = AuthUtils.GetUsername(User);
             return Ok(_teamService.GetTeamInvites(username));
         }
+
+        [HttpPatch]
+        [Route("{teamID}/accepted")]
+        public async Task<ActionResult<TeamInviteViewModel>> AcceptTeamInvite(int teamID, [FromBody] ParticipantTeamUploadViewModel acceptedInvite)
+        {
+            var username = AuthUtils.GetUsername(User);
+            if (acceptedInvite.Username != username)
```

Comment:
If you don't look up the invite from the database, this opens the door for a **security issue**. For example; I could create an object in the front end that has the id of cameron's invite, but my username instead of his. Then if I passed that object here, it would allow me to accept cameron's invite on his behalf.